### DISCUSSION TITLE
Can a Prisma Client easily expose what schema it was generated with?

### DISCUSSION THREAD

**Discussion by janpio** [prisma/prisma#6769] [Q&A]
I have a situation where I need to be able to confirm that 2 running Prisma Client instances on different servers were generated with the same Prisma Schema. Is there a simple way in Prisma Client to get information about the schema it was generated with? Maybe a hash of the schema file, that is created during the generation and embedded in the output?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
